lib/src/analyzers/lint_analyzer/rules/rules_list/prefer_extracting_callbacks/prefer_extracting_callbacks_rule.dart

```diff
@@ -6,6 +6,7 @@ import 'package:analyzer/dart/ast/visitor.dart';
 import '../../../../../utils/flutter_types_utils.dart';
 import '../../../../../utils/node_utils.dart';
 import '../../../lint_utils.dart';
+import '../../../metrics/metrics_list/source_lines_of_code/source_code_visitor.dart';
```

Comment:
Although I understand the idea of using the existing visitor, the general approach is to not import parts of modules into one another (like linter rules and metrics are separated), I'm not really sure how it should be change the best way. Maybe by moving the visitor to the common root folder?